src/index.ts

```diff
@@ -26,7 +26,7 @@ app.get("/api/books", (req, res) => {
     data: [
       { id: 1, name: "Witcher" },
       { id: 2, name: "Lord of the Rings" },
-      { id: 3, name: "Diune" },
+      { id: 3, name: "Dune" },
```

Comment:
Right 😄 You could've point it out during our session! Now it will stay on this recording forever 😄